lib/pgBackRest/Backup/Filter/PageChecksum.pm

```diff
@@ -0,0 +1,196 @@
+####################################################################################################################################
+# BACKUP FILTER PAGE CHECKSUM MODULE
+####################################################################################################################################
+package pgBackRest::Backup::Filter::PageChecksum;
+
```

Comment:
Where are we testing/exercising the checksum code? I'm having trouble finding anything that tests backups and such with checksums.